src/main/java/com/cgdpd/library/service/BookService.java

```diff
@@ -30,6 +35,20 @@ public Book createBook(CreateBookRequestDTO requestDTO) {
         return bookMapper.mapToBook(createdBook);
     }
 
+    @Transactional(readOnly = true)
+    public PagedResponse<DetailedBookDTO> getBooks(PaginationCriteria paginationCriteria,
+                                                   SearchBookCriteria searchCriteria) {
+        var pageable = paginationCriteria.toPageRequest();
```

Comment:
Give the variables accurate names, what does pageable mean here?
```suggestion
        var pageRequest = paginationCriteria.toPageRequest();
```